### ISSUE TITLE
Serialization of Models remove trailing comments

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-core#656]
when reading this model

```
Hello Xtext!
Hello Xtext2!
// comment
Hello Xtext3!
// next comment
```

and removing `Hello3` programmatically and then serializing the model the //next comment will be gone if `Hello3` is the last element in the list.

**Comment by cdietrich** [comment]
@meysholdt the resource.save does not use the new changeserializer yet and does not have a opt in for that correct?

**Comment by meysholdt** [comment]
no, resource.save() does not use the change serializer and it was never intended to do so, because the change serializer needs a recording of all changes and resource.save() can't provide that.

I think the most sustainable fix here would be to re-implement https://github.com/eclipse/xtext-core/blob/master/org.eclipse.xtext/src/org/eclipse/xtext/serializer/sequencer/HiddenTokenSequencer.java to use the text region access instead of the node model.
